shared/gh/js/views/gh.student-listview.js

```diff
@@ -13,7 +13,7 @@
  * permissions and limitations under the License.
  */
 
-define(['gh.core', 'gh.constants', 'gh.series-info'], function(gh, constants) {
+define(['gh.core', 'gh.constants', 'gh.series-info', 'clickover'], function(gh, constants) {
```

Comment:
As discussed, leaving it in `gh.student-listview` and `gh.admin-listview` since `clickover` is not known in `gh.listview`.